### PR TITLE
ARM64 work for Packet.net

### PR DESCRIPTION
**Pull request by alexellis**
## Description

ARM64 work for Packet.net

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

Port OpenFaaS Golang services to ARM64 architecture.

## How Has This Been Tested?

Manual testing - without instrumentation stack included.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by resouer** [comment]
Kindly ping to track it's status. Is this PR only waiting for tests?

File: docker-compose.arm64.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    - 'node.role == manager'
+                    - 'node.platform.os == linux'
+    prometheus:
+        image: alexellis2/prometheus:1.5.2-arm64
+        command: "-config.file=/etc/prometheus/prometheus.yml -storage.local.path=/prometheus -storage.local.memory-chunks=10000 --alertmanager.url=http://alertmanager:9093"
+        ports:
+            - 9090:9090
+        depends_on:
+            - gateway
+            - alertmanager
```
Comment (resouer) [inline docker-compose.arm64.yml]: What is alertmanager? Seems not defined.

Comment (alexellis) [reply docker-compose.arm64.yml]: alexellis2/prometheus:1.5.2-arm64 is pushed btw

**Comment by alexellis** [comment]
Merging progress.

**Pull request merged**